en/_posts/2022-07-25-news-releases.md

```diff
@@ -0,0 +1,146 @@
+- [Government of Canada will remove pre-entry test requirement for fully vaccinated travellers on April 1](https://www.canada.ca/en/public-health/news/2022/03/government-of-canada-will-remove-pre-entry-test-requirement-for-fully-vaccinated-travellers-on-april-1.html)
+
+### 2. Add temporary alerts linked to the news release on affected content pages
+
+In cases where you can’t update web content at the same time as when the news release is published, adding a temporary alert that links to a news release lets you intercept people on a content page where they naturally look for the most recent information. From there, you can direct them to “breaking news” content. 
+
+This is a stop-gap measure that can buy you some time to complete detailed updates on the content page.
+
+[Contextual alerts - Canada.ca design pattern](https://design.canada.ca/common-design-patterns/contextual-alerts.html)
+
+#### Place alerts in context, not just at the top
+
+Consider the placement of alerts carefully. An alert at the top of a page may get missed by enthusiastic scrollers. Placing it within the relevant subsection of a page lets you draw attention to an important change at the point where it makes the most sense for the reader. This maximizes the impact of the alert.
+
+#### Up your alert-content-design game
+
+Placement of alerts is only half the battle. How you design them can make a world of difference in how effective they are.
+
+- **Reading level**: Write the alert text at a grade 6 to 8 reading level. You want people to easily understand why they need to pay attention.
+- **Length**: Keep alert text very minimal. Long alerts that include too much information are distracting and disorienting. This is especially true on mobile, where they push the main content down the screen. 
+- **Effective date**: Include the exact date that changes come into effect.
+- **Verb tense**: Pay attention to your verb tense. If changes don’t go into effect for another week, don’t write in the present tense. Put your alert text in the future tense and update it once the effective date has passed. 
+
+****
+
+<figure class="gc-complex-img" role="group">
+  <img alt="A long description can be found after the image."
+src="/images/Travel-alert-1.png" >
+</figure>
+<details>
+  <summary>What to do: Clear date, correct verb tense, repeated in-context within the page</summary>
+    <p>Long description: Alert text placed at the top of the page with a clear effective date:</p>
+    <p><b>Starting April 1, 2022</b>, pre-entry tests will no longer be required for fully vaccinated travellers entering Canada by land, air, or water. Until then, follow the pre-entry test requirements. You must still use ArriveCAN before and after April 1.</p>
+    <p>News release</p>
+</details>

```

Comment:
We need to put the details/summary inside a figcaption tag inside the figure tag (see last comment)